docs/deployment/config_descriptions.md

```diff
@@ -42,10 +58,17 @@ dynamos:
 | ecs.amiid | ami-9eb4b1e5 | amiid of an optimized ecs instance (different for each region)
 | ecs.instanceType | (required) | the instance type of the ec2 machine used for running ecs tasks
 | ecs.volumeSize | 50 | the storage on ec2 instance running the ecs tasks
-| ecs.availabilityZone | us-east-1a | the availability zone used for launching ec2 machines
+| ecs.availabilityZone | us-east-1a | the availability zone used for launching ec2 machines. If using `vpc`, must match AZ of `vpc.subnets`
 | ecs.minInstances | 1 | min number of ec2 instances to launch in an autoscaling group
 | ecs.desiredInstances | 1 | desired number of ec2 instances needed in an autoscaling group
 | ecs.maxInstances | 2 | max number of ec2 instances to launch in an autoscaling group
+| ecs.publicIp | false | assign public IP(s) to ECS instance(s)
+| ecs.restartTasksOnDeploy | false | whether kes should restart ECS tasks on deploy
+| ecs.docker.username | (empty string) | Username to pull images from registry
+| ecs.docker.registry | dockerhub | container registry to fetch from
+| ecs.services | | ECS service configurations
```

Comment:
These configuration blocks are fairly involved. Should we just point people to the deployment package for an example of how to use them?

Reply:
👍 632baf9